.github/workflows/release_parallel.yml

```diff
@@ -0,0 +1,218 @@
+name: Parallel Release
+on:
+    workflow_call:
+        secrets:
+            GPG_SIGNING_KEY:
+                required: true
+            GH_TOKEN:
+                required: true
+            NPM_TOKEN:
+                required: true
+            LASTMJS_GITHUB_TOKEN:
+                required: true
+
+jobs:
+    prepare-release:
+        name: Prepare Release
+        runs-on: ubuntu-latest
+        env:
+            GPG_SIGNING_KEY: ${{ secrets.GPG_SIGNING_KEY }} # All commits must be verified
+            GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Was this token used here?

Reply:
We have a number of env variables that even if they are not used directly in the yml are used in the tests or else where. It is hard to see just by looking at the yml if this token is used by us, or by an script or something we rely on. We can test a work on getting rid of these relics but it felt outside the scope of this job